librad/src/git/storage.rs

```diff
@@ -992,6 +992,14 @@ where
 
         Ok(())
     }
+
+    pub fn is_tracked(&self, urn: &RadUrn, peer: &PeerId) -> Result<bool, Error> {
```

Comment:
It would be great to document new functions :pray:

Reply:
Why would be catched with a `#[warn(missing_docs)]` and `--D warnings` in the CI run.

Reply:
That's a bigger effort though, currently.

Reply:
#302